deliravision/torch/models/backbones/unet.py

```diff
@@ -289,18 +293,20 @@ def __init__(self, n_dim, in_channels, out_channels,
                     self.conv1 = conv3x3(self.n_dim,
                                          out_channels,
                                          self.out_channels)
+                self.norm1 = NormNdTorch(norm_layer, n_dim, self.out_channels)
                 self.conv2 = conv3x3(self.n_dim,
                                      self.out_channels,
                                      self.out_channels)
+                self.norm2 = NormNdTorch(norm_layer, n_dim, self.out_channels)
 
             def forward(self, from_down, from_up):
                 from_up = self.upconv(from_up)
                 if self.merge_mode == 'concat':
                     x = torch.cat((from_up, from_down), 1)
                 else:
                     x = from_up + from_down
-                x = F.relu(self.conv1(x))
-                x = F.relu(self.conv2(x))
+                x = F.relu(self.norm1(self.conv1(x)))
```

Comment:
Does our norm wrapper support no norm at all?

Reply:
No, currently we do not support  no normalization. But I think we should support it (I just was not shure if I should add this to this pr, but it should be a very minor change)

Reply:
I think you can include it in this PR, since this only is a minor change